src/adapter/src/catalog/builtin_table_updates.rs

```diff
@@ -57,6 +63,10 @@ use crate::catalog::{
 use crate::session::Session;
 use crate::subscribe::ActiveSubscribe;
 
+use super::builtin::{
+    MZ_PREPARED_STATEMENT_HISTORY, MZ_SESSION_HISTORY, MZ_STATEMENT_EXECUTION_HISTORY,
+};
```

Comment:
Nit: The import style says to prefer `crate::` paths before `super::` paths: https://github.com/MaterializeInc/materialize/blob/main/doc/developer/style.md#imports